docs/guides/upgrading.rst

```diff
@@ -3,12 +3,22 @@ Upgrading Plugins
 
 Prepare your plugin for the next version of Elgg.
 
-See the administator guides for :doc:`how to upgrade a live site </admin/upgrading>`.
+See the administrator guides for :doc:`how to upgrade a live site </admin/upgrading>`.
 
 .. contents:: Contents
    :local:
    :depth: 2
 
+From 1.11 to 2.0
+================
+
+Breadcrumb display now removes the last item if it does not contain a link. If you need the previous behavior,
+unregister the handler ``elgg_prepare_breadcrumbs`` for the plugin hook ``'prepare', 'breadcrumbs'`` and replace
+it with a custom handler that only applies the excerpt.
```

Comment:
"A custom handler that only applied the excerpt" is not super clear to me. Can you also include a code example?